desafio4.sql

```diff
@@ -0,0 +1,13 @@
+-- DROP VIEW top_3_artistas;
```

Comment:
@juniomelos eu pesquisei e achei um jeito de executar sem precisar usar o DROP separado ou deixar comentado.
Acho que isso pode vir a ser útil pra você:

```suggestion
-- DROP VIEW  IF EXISTS top_3_artistas;
```
Vi em:
https://www.mysqltutorial.org/mysql-views/mysql-drop-view/

Reply:
Não sei se passaria no teste tbm :thinking: